www/src/layouts/index.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import PluginLibraryWrapper from "../components/layout/plugin-library-wrapper"
+
+export default props => {
+  if (props.pageContext.layout === `plugins`) {
+    return (
+      <PluginLibraryWrapper location={props.location}>
+        {props.children}
+      </PluginLibraryWrapper>
+    )
+  }
+  return <div>{props.children}</div>
```

Comment:
Is a `<React.Fragment>` also possible here? I'm always for minimizing the `div`-ception

Reply:
Yeah, good idea.